internal/config/config.go

```diff
@@ -113,6 +113,7 @@ type SQLGen struct {
 type SQLGo struct {
 	EmitInterface       bool              `json:"emit_interface" yaml:"emit_interface"`
 	EmitJSONTags        bool              `json:"emit_json_tags" yaml:"emit_json_tags"`
+	EmitDBTags          bool              `json:"emit_db_tags" yaml:"emit_db_tags"`
```

Comment:
[sqlx](https://github.com/jmoiron/sqlx#usage) also uses the `db` tag for annotating structs. Can we change this value to `emit_gorp_tags`? I want to make sure we have the flexibility to support other tools. This means we'd need to change the name of the test folder as well.

Reply:
sqlx uses the `db` tag in the same way though, no? (I guess this depends on the resolution to that that other comment thread).

Reply:
I don't think sqlx has support for the autoincrement and primary key annotations? We'd need to confirm with both library authors that the annotations are compatible with each other